<ui version="4.0" >
 <class>CubeForm</class>
 <widget class="QDialog" name="CubeForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Create Cube</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>30</x>
     <y>110</y>
     <width>351</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>40</y>
     <width>71</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cube Size</string>
   </property>
   <property name="buddy" >
    <cstring>sizeField</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="sizeField" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>40</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="sizeSlider" >
   <property name="geometry" >
    <rect>
     <x>220</x>
     <y>40</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum" >
    <number>-1000</number>
   </property>
   <property name="maximum" >
    <number>1000</number>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>sizeField</tabstop>
  <tabstop>sizeSlider</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CubeForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CubeForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
